editor/plugins/version_control_editor_plugin.cpp

```diff
@@ -67,6 +67,7 @@ void VersionControlEditorPlugin::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("_ssh_private_key_selected"), &VersionControlEditorPlugin::_ssh_private_key_selected);
 	ClassDB::bind_method(D_METHOD("_commit_message_gui_input"), &VersionControlEditorPlugin::_commit_message_gui_input);
 	ClassDB::bind_method(D_METHOD("_cell_button_pressed"), &VersionControlEditorPlugin::_cell_button_pressed);
+	ClassDB::bind_method(D_METHOD("_confirm_discard_all"), &VersionControlEditorPlugin::_confirm_discard_all);
```

Comment:
You don't need to bind the method anymore in 4.0 since you're using `callable_mp`, so it's using the method pointer directly instead of going through ClassDB. For this kind of internal callback, you only need to bind the ones that are still call via method bindings (typically they're passed around as strings, e.g. `undo_redo->add_do_method(this, "_redraw_tracks");` requires `_redraw_tracks` to be bound).

Might be worth reviewing the other bindings too.